ui/src/pages/Deployment.vue

```diff
@@ -37,11 +37,7 @@
       </template>
 
       <template #runs>
-        <FlowRunFilteredList :flow-run-filter="deploymentFilter">
-          <template #empty-message>
-            No runs found
-          </template>
-        </FlowRunFilteredList>
+        <FlowRunFilteredList :flow-run-filter="deploymentFilter" />
```

Comment:
Can you make one additional change here and use this same component on the `FlowRuns.vue` page as well? I think we're potentially using the wrong one there and it's actually not loading more data like this one does.

Reply:
So I agree we need to address that. But I don't want to do that in this PR. Mainly because the flow runs page has a refresh interval. Which we haven't implemented in the `FlowRunFilteredList` or anywhere we use list paging that I'm aware of.